gke-tpu-examples/single-host-inference/jax/stable-diffusion/serve-stable-diffusion-v4.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: apps/v1
```

Comment:
should we rename the files to have a consistent naming (tpu-v4 and tpu-v5 suffixes)

Reply:
Renamed the file to serve-stable-diffusion-tpu-v4.yaml here and in the doc as suggested.

Reply:
can we rename the existing file as well to conform to the same naming format?

Reply:
It's a little tricky since the existing files are used by other docs that have been published. On the safe side, I'd say to leave them as is.